docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -18,13 +18,13 @@ For example, `0 0 0 * * ?` runs every 24 hrs at 12:00 a.m. `0 0 1 * * ?` runs ev
 If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.
 . Click on the edit icon and click *Add Node*.
 . Set up your requisition as described in xref:operation:provisioning/getting-started.adoc#requisition-create[Create a requisition].
-. Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`.
+. Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`. For example to specify a "running" configuration (i.e. configuration that the device currently has while running), create a service with the name `DeviceConfig-running`. This will display in the UI as "Running Configuration". If you specify `DeviceConfig-running-xml`, this will display in the UI as "Running Xml Configuration".
 . Click *Add Meta-Data*, and define the following key-value pairs:
-.. `dcb:username`: username for the device
-.. `dcb:password`: password for the device
-.. `dcb:script-file`: name of the device configuration script for this type of device without .dcb extension
+.. `dcb:username`: username for the device. This can also be an SCV metadata expression.
```

Comment:
Make "SCV metadata expressions" a link to the SCV metadata expressions documentation.
Will need to update the /operation/pages/meta-data.adoc file to add an anchor tag to the SCV topic, then link to it. (I think this is the right link, unless there is other SCV documentation that is more applicable.)